README.md

```diff
@@ -123,6 +123,7 @@ Spiderable.debug = true
  - Create template which you prefer to return, when page is not found
  - Set `Spiderable.customQuery`
  - Set iron router's `notFoundTemplate`
+ - Include a comment '<!-- response:status-code=404 -->' on your template. This way, we can ensure spiderable sends a 404 status code in the response headers
```

Comment:
Good observation. The interesting fact is that on js-rendered pages the comments are removed. But on the phantomjs rendered ones, they are not.

Have a look at https://4scotty.com/404?_escaped_fragment_= - status code is 404 and the comment is visible in the source